connectors/sources/confluence.py

```diff
@@ -304,6 +312,34 @@ def tweak_bulk_options(self, options):
         """
         options["concurrent_downloads"] = self.concurrent_downloads
 
+    async def validate_config(self):
+        """Validates whether user input is empty or not for configuration fields.
+        Also validate, if user configured spaces are available in Confluence.
+
+        Raises:
+            Exception: Configured keys can't be empty
+        """
+        self.configuration.check_valid()
+        await self._remote_validation()
+
+    async def _remote_validation(self):
```

Comment:
I think we should add `@retryable` to the remove validation to have consistent behaviour across the different connectors.

Reply:
Makes sense, missed that `api_call` already uses `retryable`.